plop-templates/package/package.json.hbs

```diff
@@ -1,12 +1,21 @@
 {
   "name": "pcln-{{kebabCase componentName}}",
   "version": "0.0.1",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
   "scripts": {
-    "prepare": "babel src -d dist",
-    "clean": "rm -rf node_modules dist",
+    "prepare": "npm run clean && npm run build",
```

Comment:
N-b; could be `npm-run-all clean build`

Reply:
I've pushed this change. Please resolve if it is satisfactory.